app/src/evaluation/literalai_exporter.py

```diff
@@ -46,6 +46,41 @@ def lai_link(self) -> str:
             f"threads/{self.thread_id}?currentStepId={self.question_id}"
         )
 
+    @classmethod
+    def csv_dict_headers(cls) -> list[str]:
+        return [
+            "User ID",
+            "Date",
+            "Question",
+            "Response",
+            "LiteralAI Thread",
+            "Agency ID",
+            "Session ID",
+            "Program",
+            "Citation Links",
+            "Citation Sources",
+            "Has Chat History",
+            "Thread ID",
+            "Timestamp",
+        ]
```

Comment:
I would extract these names from the `to_csv_dict()` to avoid any misalignment/typos.